compiler/daml-lf-verify/BUILD.bazel

```diff
@@ -46,7 +46,7 @@ da_haskell_binary(
         "//libs-haskell/bazel-runfiles",
         "//libs-haskell/da-hs-base",
     ],
-)
+) if not is_windows else None
```

Comment:
Why do we need to disable everything on Windows? I get that we need to disable the tests until we get Z3 from somewhere but building things seems like it should be fine?

Reply:
That's a fair point. Fixed, now only the tests are disabled :)